### PR TITLE
fix(fhir): upd condition resource

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport-internal#1441

### Description

- Updated the Condition resource to properly render the clinicalStatus and category attributes

### Testing

_[Regular PRs:]_

- Local
  - [x] Checked the presence of these new attributes in the output
  - [x] Checked that the length of the json output increased

### Release Plan
- Merge this

### CODE REVIEW

File: packages/fhir-converter/src/templates/cda/Resources/Condition.hbs [hunk 1]
```diff
@@ -12,10 +12,12 @@
         "clinicalStatus":
         [ 
             {{#each (toArray conditionEntry.entryRelationship)}}
-                {{#if (eq this.observation.templateId.root '2.16.840.1.113883.10.20.22.4.5')}} 
-                    {{>DataType/CodeableConcept.hbs code=this.observation.value}},
-                {{/if}}
-          {{/each}}
+                {{#each (toArray this.observation.templateId)}}
+                    {{#if (eq this.root '2.16.840.1.113883.10.20.22.4.6')}}
```
Comment (RamilGaripov) [inline packages/fhir-converter/src/templates/cda/Resources/Condition.hbs:16]: note, the `2.16.840.1.113883.10.20.22.4.5` had a one-digit typo, never evaluating to true

File: packages/fhir-converter/src/templates/cda/Resources/Condition.hbs [hunk 2]
```diff
@@ -30,6 +32,12 @@
                 {{/each}}
             ],
         {{/if}}
+
+        "category":{{>DataType/CodeableConcept.hbs code=conditionEntry.entryRelationship.observation.code}},
```
Comment (Goncharo) [inline packages/fhir-converter/src/templates/cda/Resources/Condition.hbs:36]: do we know `conditionEntry.entryRelationship.observation.code` will always be defined?

maybe we should put this behind an `if`?

File: packages/fhir-converter/src/templates/cda/Resources/Condition.hbs [hunk 3]
```diff
@@ -12,10 +12,12 @@
         "clinicalStatus":
         [ 
             {{#each (toArray conditionEntry.entryRelationship)}}
-                {{#if (eq this.observation.templateId.root '2.16.840.1.113883.10.20.22.4.5')}} 
-                    {{>DataType/CodeableConcept.hbs code=this.observation.value}},
-                {{/if}}
-          {{/each}}
+                {{#each (toArray this.observation.templateId)}}
+                    {{#if (eq this.root '2.16.840.1.113883.10.20.22.4.6')}} 
+                        {{>DataType/CodeableConcept.hbs code=../this.observation.value}},
```
Comment (Goncharo) [inline packages/fhir-converter/src/templates/cda/Resources/Condition.hbs:17]: was adding the `../` intentional?

File: packages/fhir-converter/src/templates/cda/Resources/Condition.hbs (hunk 2 above)
Comment (RamilGaripov) [reply packages/fhir-converter/src/templates/cda/Resources/Condition.hbs:36]: I noticed that when it's not present, the "category" attribute just doesn't get inserted. So the `if` block seems to be redundant

File: packages/fhir-converter/src/templates/cda/Resources/Condition.hbs (hunk 3 above)
Comment (RamilGaripov) [reply packages/fhir-converter/src/templates/cda/Resources/Condition.hbs:17]: ya, this is to change the context for `this` from the inner loop to the outer loop

Comment (Goncharo) [reply packages/fhir-converter/src/templates/cda/Resources/Condition.hbs:17]: didn't know you could do this, cool

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**